bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/thing/channel.xml

```diff
@@ -56,4 +57,18 @@
 		<category>ColorLight</category>
 	</channel-type>
 
+	<channel-type id="effect">
+		<item-type>Number</item-type>
```

Comment:
Please change this to a String item-type channel so users can use meaningful names in their .things files instead of numbers.

```suggestion
		<item-type>String</item-type>
```

Strings will also make it easier for translating the values. Adding an enum for these values would make sense so  it can be used to map a value to a LIFX constant.